src/trumbowyg.js

```diff
@@ -77,6 +77,12 @@ Object.defineProperty(jQuery.trumbowyg, 'defaultOptions', {
         imageWidthModalEdit: false,
 
         prefix: 'trumbowyg-',
+// classes for inputs
+        h1Class: null,
+        h2Class: null,
+        h3Class: null,
+        h4Class: null,
+        pClass: null,
```

Comment:
Maybe it will be better to have an object like that:

```
tagClasses: {}
```

Which will allow adding class to more tags :)

Reply:
is done